relay-server/src/services/processor.rs

```diff
@@ -2251,6 +2262,13 @@ impl UpstreamRequest for SendEnvelope {
     }
 }
 
+/// Returns a boolean whether the current item should be sampled or discarded
```

Comment:
nit: Didn't find a reference for this but this seems to be the Rust way.
```suggestion
/// Returns `true` if the current item should be sampled.
```